components/profile/time.rs

```diff
@@ -300,16 +299,17 @@ impl Profiler {
     }
 
     /// Get tuple (mean, median, min, max) for profiler statistics.
-    pub fn get_statistics(data: &[f64]) -> (f64, f64, f64, f64) {
-        data.iter().fold(-f64::INFINITY, |a, &b| {
+    pub fn get_statistics(data: &[Duration]) -> (Duration, Duration, Duration, Duration) {
+        data.iter().fold(0, |a, &b| {
+            let b = b.whole_nanoseconds();
```

Comment:
This could be simplified even further, I think since it is only trying to make sure the data is sorted:

```rust
        debug_assert!(
            data.windows(2).all(|window| window[0] <= window[1]),
            "Data must be sorted"
        );
```

I've done this.

Reply:
Nice!